TRANSLATING.md

```diff
@@ -0,0 +1,74 @@
+# 🌐 <abbr title="translation">Translation</abbr> Guide
+
+Table of contents:
```

Comment:
It should autogenerate this in the sidebar, no? Not sure we need it manually as well?

Reply:
You're right. I forgot of this GitHub feature 😬